File: read_excel.js
const xlsx = require('xlsx');
const fs = require('fs');

try {
    const workbook = xlsx.readFile('C:\\Users\\gunas\\Desktop\\M1g\\EKİP LİSTE ŞABLOM.xlsx');
    const sheetName = workbook.SheetNames[0];
    const sheet = workbook.Sheets[sheetName];
    const data = xlsx.utils.sheet_to_json(sheet);
    
    console.log("COLUMNS:");
    if (data.length > 0) {
        console.log(Object.keys(data[0]));
    }
    console.log("\nFIRST 5 ROWS:");
    console.log(JSON.stringify(data.slice(0, 5), null, 2));
} catch (e) {
    console.error("Error reading file:", e.message);
}

File: read_excel.py
import pandas as pd
import json

try:
    df = pd.read_excel(r'C:\Users\gunas\Desktop\M1g\EKİP LİSTE ŞABLOM.xlsx')
    # Print the columns
    print("COLUMNS:")
    print(df.columns.tolist())
    # Print the first 5 rows
    print("\nFIRST 5 ROWS:")
    print(df.head(5).to_json(orient='records', force_ascii=False, indent=2))
except Exception as e:
    print("Error:", e)
